.github/workflows/auto-publish.this_should_be_yml

```diff
@@ -0,0 +1,19 @@
+# .github/workflows/pr-push.yml
+name: CI
+on:
+  pull_request: {}
+  push:
+    branches: [main]
+jobs:
+  main:
+    name: Build, Validate and Deploy
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: w3c/spec-prod@v2
+        with:
+          W3C_ECHIDNA_TOKEN: ${{ secrets.W3C_TR_TOKEN }}
+          W3C_WG_DECISION_URL: https://www.w3.org/publishing/groups/publ-wg/Meetings/Minutes/2019/2019-08-19-pwg#resolution3
+          W3C_BUILD_OVERRIDE: |
+             shortName: pub-manifest
+             specStatus: REC
```

Comment:
This is fine to leave as-is until we actually supports REC publications through Echidna.

Reply:
Yes, again, this is to be updated in case we reopen the wg...